tests/test_algorithms.py

```diff
@@ -83,3 +83,51 @@ def test__split_tests_calculates_avg_test_duration_only_on_present_tests(self, a
         expected_first, expected_second = expected
         assert first.selected == expected_first
         assert second.selected == expected_second
+
+    def test_each_group_is_assigned_a_test_front_loaded(self):
+        from collections import namedtuple
+
+        from pytest_split import algorithms
+
+        item = namedtuple("item", "nodeid")
+
+        durations = {}
+        durations["a"] = 100
+        durations["b"] = 1
+        durations["c"] = 1
+        durations["d"] = 1
+        durations["e"] = 1
+        durations["f"] = 1
+        durations["g"] = 1
+        durations["h"] = 1
+
+        items = [item(x) for x in ["a", "b", "c", "d", "e", "f", "g", "h"]]
+
+        groups = algorithms.duration_based_chunks(7, items, durations)
```

Comment:
Why the first arg is 7 here and 8 in the next test?

Reply:
typo; fixed 👍